### ISSUE TITLE
Heavy dependency - intl through league/uri

### ISSUE THREAD

**Issue by joeyhub** [docker-php/docker-php#302]
I can't quite imagine what you might be doing that needs intl.

It's coming in through:

 - league/uri 4.2.3 requires ext-intl * -> the requested PHP extension intl is missing from your system.

intl doesn't seem like something you're likely to need for your case or most cases of uri handling.

**Comment by joelwurtz** [comment]
Yes it's need by league uri, which is needed by jane which is needed by me. Not sure if easily resolvable, may update jane to get right of this deps but clearly not a priority.
